### ISSUE TITLE
Bug tfc uses python2 in the cloud container

### ISSUE THREAD

**Issue by haifeng-jin** [tensorflow/cloud#38]
When I have tf version 2.1.0 installed locally with python3 interpreter, not providing the docker_base_image arg to tfc.
tfc would use python2.7 in the cloud container.
`File "/usr/local/lib/python2.7/dist-packages/kerastuner/engine/hypermodel.py`
[log link](https://pantheon.corp.google.com/logs/viewer?resource=ml_job%2Fjob_id%2Ftf_cloud_train_2fcde103_0d9e_481d_8076_c11023d64bd4&interval=NO_LIMIT&project=keras-team-gcp&minLogLevel=0&expandAll=false&timestamp=2020-07-07T21:20:47.726000000Z&customFacets=&limitCustomFacetWidth=true&scrollTimestamp=2020-07-07T21:16:31.940949494Z&advancedFilter=resource.labels.job_id%3D%22tf_cloud_train_2fcde103_0d9e_481d_8076_c11023d64bd4%22%20timestamp%3E%3D%222020-07-07T21:13:16Z%22&dateRangeUnbound=forwardInTime)

**Comment by pavithrasv** [comment]
This has been fixed in https://github.com/tensorflow/cloud/pull/35. Please feel free to reopen issue if you are continuing to see this problem.
